jinahub/rankers/MinRanker/minranker.py

```diff
@@ -0,0 +1,52 @@
+__copyright__ = "Copyright (c) 2020-2021 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+from itertools import groupby
+from typing import List, Dict
+
+from jina import Executor, requests, DocumentArray
+from jina.logging.logger import JinaLogger
+
+
+class MinRanker(Executor):
+    """
+    :class:`MinRanker` aggregates the score
+    of the matched doc from the matched chunks.
+    For each matched doc, the score is aggregated
+    from all the matched chunks belonging to that doc.
+    :param metric: the distance metric used in `scores`
```

Comment:
Maybe format this to look nicer - no need to break lines so early, and there should be an empty line between description and args